draft-barnes-acme.md

```diff
@@ -1398,46 +1398,81 @@ If the server is able to validate the signature and confirm that the jwk and alg
 
 ## DNS
 
-When the identifier being validated is a domain name, the client can prove control of that domain by provisioning records under it.   The DNS challenge requires the client to provision a TXT record containing a validation token under a specific validation domain name.
+When the identifier being validated is a domain name, the client can prove control of that domain by provisioning resource records under it.   The DNS challenge requires the client to provision a TXT record containing a designated value under a specific validation domain name.
 
 type (required, string):
 : The string "dns"
 
 token (required, string):
-: An ASCII string that is to be provisioned in the TXT record.  This string SHOULD be randomly generated, with at least 128 bits of entropy (e.g., a hex-encoded random octet string).
+: A random value unique among challenges issued by this server.  This value MUST have at least 128 bits of entropy.  It MUST NOT contain any characters outside the URL-safe Base64 alphabet.
```

Comment:
Instead of saying it's "unique" why not say it must be 160 bits and cryptographically random.